config/locales/en.yml

```diff
@@ -17,7 +17,7 @@ en:
     school_page_link: 'My jobs'
   sign_in:
     title: 'Sign in to Teaching Jobs'
-    intro: 'While the service is in development it lists teaching jobs in select areas, currently Cambridgeshire, the North East and Milton Keynes. In order for you to sign into the service we need to know which area your school is in. '
+    intro: 'While the service is in development it only lists teaching jobs in the following areas. In order for you to sign into the service we need to know which area your school is in. '
```

Comment:
The copy on the PR doesn't read very well if MK is only removed but no further amendments take place.

`While the service is in development it only lists jobs in the following areas. In order for you to sign into the service we need to know which area  your school is in.`

in the following areas... reads as if it's missing some continuation

Should this be split into two and and the first part be inset text (similar to here https://govuk-elements.herokuapp.com/typography/#typography-inset-text ) so it's highlited?

e.g.

> While the service is in development it only lists jobs in the areas listed below.

In order for you to sign into the service we need to know which area  your school is in.